PhysicsTools/Heppy/python/analyzers/objects/LeptonAnalyzer.py

```diff
@@ -187,9 +194,24 @@ def makeLeptons(self, event):
             if self.miniIsolationVetoLeptons == "inclusive":
                 for lep in event.inclusiveLeptons: 
                     self.IsolationComputer.addVetos(lep.physObj)
+            event.LeptonTrackMuPairs = []
             for lep in event.inclusiveLeptons:
                 self.attachMiniIsolation(lep)
-        
+                if self.doDirectionalIsolation:
+                    for cs in xrange(2,7): self.attachDirectionalIsolation(lep,0.1*cs)
```

Comment:
Maybe it would be better to have `doDirectionalIsolation` be equal to the list of cone sizes to compute, so that we don't recompute 5 isolation cones every time?